plugins/org.komodo.importer/src/org/komodo/importer/vdb/VdbImporter.java

```diff
@@ -52,62 +55,90 @@ public VdbImporter(Repository repository) {
     }
 
     @Override
-    protected KomodoObject executeImport(UnitOfWork transaction,
+    protected void executeImport(UnitOfWork transaction,
                                                                      String content,
+                                                                     KomodoObject parentObject,
                                                                      ImportOptions importOptions,
                                                                      ImportMessages importMessages) throws KException {
 
         String vdbName = importOptions.getOption(OptionKeys.NAME).toString();
         String vdbFilePath = importOptions.getOption(OptionKeys.VDB_FILE_PATH).toString();
-        KomodoObject workspace = getWorkspace(transaction);
 
-        Vdb vdb = getWorkspaceManager().createVdb(transaction, workspace, vdbName, vdbFilePath);
+        Vdb vdb = getWorkspaceManager().createVdb(transaction, parentObject, vdbName, vdbFilePath);
         KomodoObject fileNode = vdb.addChild(transaction, JcrLexicon.CONTENT.getString(), null);
         fileNode.setProperty(transaction, JcrLexicon.DATA.getString(), content);
-        return vdb;
     }
 
+    @Override
+    protected boolean handleExistingNode(UnitOfWork transaction,
+    		ImportOptions importOptions,
+    		ImportMessages importMessages) throws KException {
+    	
+    	// VDB name to create
+    	String vdbName = importOptions.getOption(OptionKeys.NAME).toString();
+
```

Comment:
Why has this logic been moved down from AbstractImporter when it could equally apply to a schema node. The CREATE_NEW may no longer be applicable but the OVERWRITE certainly would be since it would be essential to remove the node / node's children prior to sequencing.